### ISSUE TITLE
Tooltips are not dismissible

### ISSUE THREAD

**Issue by Amnish04** [ng-lightning/ng-lightning#600]
According to [WCAG2.1](https://www.w3.org/TR/WCAG21/) standard - [Content on hover or focus](https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html), any additional content that appears on either pointer hover or keyboard focus should be hoverable, persistent and **dismissible**.

![Content on hover or focus description/definition](https://github.com/ng-lightning/ng-lightning/assets/78865303/62c354ba-c76a-4d4b-b381-6e65dee670df)

While `nglTooltip` passes the first 2 criteria, it unfortunately does not comply with the last one i.e., it **cannot be dismissed** using a keyboard combination (at least based on my knowledge and the docs).

We are using `ng-lightning` in an enterprise project and need to comply with all the WCAG standards by a specific deadline, which is why it would be great if this issue could be resolved in the upstream itself.

## More details
ng-lighting version: `^9.0.0`
Angular version     : `^16.2.12`
